.travis.yml

```diff
@@ -11,7 +11,6 @@ cache:
     - $HOME/.cache # includes bower's cache
 
 env:
-  - EMBER_TRY_SCENARIO=ember-1.13
```

Comment:
Correct. `1.4.0` of tether is required to set the equivalent of `rootElement` on the tether library.